specification/tags.json

```diff
@@ -138,6 +138,10 @@
     "name": "ReturnMethods",
     "externalDocs": {}
   },
+  {
+    "name": "ReturnPaymentRequests",
+    "externalDocs": {}
+  },
```

Comment:
That will also remove the need for another header (which we would need to hide)
```suggestion
```

Reply:
done